crates/workspace/src/workspace.rs

```diff
@@ -6474,3 +6482,176 @@ mod tests {
         });
     }
 }
+
+pub fn client_side_decorations(element: impl IntoElement, cx: &mut WindowContext) -> Stateful<Div> {
+    const ROUNDING: Pixels = px(10.0);
+    const SHADOW_SIZE: Pixels = px(10.0);
+    const BORDER_SIZE: Pixels = px(1.0);
+    let grey = rgb(0xC8C8C8);
```

Comment:
```suggestion
    let color = cx.theme().colors().border;
```
I think this looks much better than hard-coding a greyish color :slightly_smiling_face:

![image](https://github.com/zed-industries/zed/assets/71973804/ca7c7f70-0aff-4b7a-820e-06655c24044e)

Also, there seem to still be some clipping issues with the two rounded window corners:
![image](https://github.com/zed-industries/zed/assets/71973804/1252be8b-f266-4d12-854d-ba271cebd84c)